component/site/libraries/helper.php

```diff
@@ -1670,67 +1671,67 @@ function canPublishOwnEvents($evid, $vevent = false)
 			{
 				return true;
 			}
-                        
```

Comment:
I've updated my NetBeans code style settings to more closely match the Joomla standards so hopefully this issue will diminish in the future.  Just checking the change now.